packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -1313,7 +1313,10 @@ export class DatePicker {
     }
 
     return (
-      <Host onKeyDown={this.keyDownHandler} class={size}>
+      <Host
+        onKeyDown={this.keyDownHandler}
+        class={{ [`ic-date-picker-${size}`]: true }}
```

Comment:
```suggestion
        class={`ic-date-picker-${size}`}
```

slightly more succinct for just a single class

Reply:
It comes up with "property assignment expected" for this line if I remove the square brackets. Might be something to do with the back ticks